src/LearnositySdk/Request/Init.php

```diff
@@ -118,10 +118,11 @@ public function __construct($service, $securityPacket, $secret, $requestPacket =
             $requestString = $requestPacket;
```

Comment:
We can remove this, as we overwrite it below.

Reply:
Yes - I actually did that in docs site sdk which I'm using to play with but haven't pushed it here. Cheers

Reply:
@shtrom this got re-added, because we don't want to re-generate `$requestString` if request was passed as string and user chose not to add meta to the request. The reason for `$addMeta` parameter is explained below in another comment.